independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test/simple/SimpleQuarkusRestResource.java

```diff
@@ -75,8 +75,10 @@ public String get(@PathParam("id") String id) {
     public String params(@PathParam("p") String p,
             @QueryParam("q") String q,
             @HeaderParam("h") int h,
+            @HeaderParam("h2") char h2,
+            @HeaderParam("h3") Character h3,
             @FormParam("f") String f) {
```

Comment:
I know the other primitive types work, because I have tests for them, but I'm surprised that we don't here.

Reply:
?

Reply:
I mean `boolean`, `byte`, `short`, `long`, `float`, `double` and all that junk.

Reply:
We could add them, but there is not much point I would say due the existence of `valueOf`